libs/labelbox/src/labelbox/data/annotation_types/label.py

```diff
@@ -85,6 +88,29 @@ def frame_annotations(
                 frame_dict[annotation.frame].append(annotation)
         return frame_dict
 
+    def audio_annotations_by_frame(
+        self,
+    ) -> Dict[
+        int, List[Union[AudioObjectAnnotation, AudioClassificationAnnotation]]
+    ]:
```

Comment:
Remove `audio_annotations_by_frame` and add the `AudioClassificationAnnotation` and `AudioTextClassificationAnnotation` to the `frame_annotations()` method